Cargo.toml

```diff
@@ -4,6 +4,7 @@
 [workspace]
 members = [
     "rust/shlex",
+    "rust/module",
```

Comment:
I feel like this could use a slightly more distinctive name.

Reply:
I see what you mean and I'd definitely agree outside of the kernel, but in kernel context "module" refers to that (e.g. the `M`/`m` in configs, `MODULE` as a def, `module.h` as an include...), so using `kernel_module` seems too verbose.

In general, I don't think we should put a `kernel_*` prefix to most things. For instance, the `KernelResult` type is a bit verbose too. In cases like that where their names conflict with something (e.g. with `core::result::Result`) we could have a `K` prefix, e.g. `KResult` (the `K`/`k` prefix/suffix are also used for this in the C side in many places).